types/node/index.d.ts

```diff
@@ -35,6 +35,7 @@
 //                 Jeremie Rodriguez <https://github.com/jeremiergz>
 //                 Samuel Ainsworth <https://github.com/samuela>
 //                 Kyle Uehlein <https://github.com/kuehlein>
+//                 Thanik Bhongbhibhat <https://github.com/bhongy>
```

Comment:
Seems like this section used to be alphabetical. Happy to put my name in alphabetically instead.

Reply:
We're going to sort it eventually 🙂